documentation/develop/build-an-accessible-extension.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: sidebar
+title: Build an accessible extension
+permalink: /documentation/develop/build-an-accessible-extension/
+published: false
+topic: Develop
+tags: [Development, Extensions, UI, User Interface, UX, Web Extensions]
+contributors: [rebloor]
+last_updated_by: rebloor
+date: 2019-05-01 9:08:50
+---
+
+<!-- Page Hero Banner -->
+
+<section class="page-hero">
```

Comment:
Can all of these be includes too all with all the other "structural" markup used for the layout in each document. Since they're duplicated all over the place it would be a lot nicer to include them and have one instance to update in the event of future changes.

Reply:
Oh yeah, that's a very good idea. It would make the pages easier to read as well.